net/teleport/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        gravitational teleport 2.0.6 v
+revision            0
+
+homepage            http://gravitational.com/teleport/
+categories          net
+maintainers         gclawes
+description         Teleport SSH server and Certificate Authority
+long_description    Teleport is a modern SSH server and CA for managing clusters \
+                    that provides secure login with certificate-based access, \
+                    2-factor authentication, session logging and replay, \
+                    integration with Google Apps and other OAuth identity providers, \
+                    and a Web UI.  Built on the Golang SSH library, and compatible with OpenSSH \
+license             Apache2
+
+checksums           rmd160  19d99162dbde0190ddadde27c855b9c21a716c8b \
+                    sha256 f6211d5a52d5b3806f44ae5ef3bda349d1c6924961ba70b953148ad9e793f3f1
+
+set goproj          github.com/${github.author}/${github.project}
+
+depends_lib         port:go port:zip
+platforms           darwin
+use_configure       no
+worksrcdir          src/${goproj}
+
+post-extract {
+    xinstall -d ${workpath}/src/github.com/${github.author}
+    move ${workpath}/${name}-${github.version} \
+        ${worksrcpath}
+}
+
+build {
+    system -W ${worksrcpath} "GOPATH=${workpath} make release"
```

Comment:
Fixed: 2772677312caeda829b3f7ee75323537096dfdd2

Had to disable parallel build for the zip call to work.